### DISCUSSION TITLE
mathjax 사용

### DISCUSSION THREAD

**Discussion by qkrwoghd04** [pronist/hELLO#203] [Q&A] [answered]
mathjax 사용을 위해서 아래와 같은 코드를 HTML에 넣으면,

<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>

미리보기에서는 노출이 되지만, 포스팅에서는 노출이 안됩니다
이것저것 다 찾아봤는데, https://pronist.dev/5#FAQ-1에 올리신 <script>를 삽입하고싶어요
글도 읽어봤는데, 왜 그런지는 알겠는데,
어떻게 처리해야하는지 잘모르겠습니다...

**Comment by qkrwoghd04** [comment, marked as answer]
해결 했습니다!

**Reply by Rdtddt** [reply]
혹시 이거 어떻게 해결하셨나요? 저도 지금 이거때문에 고민중입니다...

**Reply by GreenScreen410** [reply]
@Rdtddt 저의 경우는 명확한 해결 방법을 찾지 못하여 테마 v4.0.0 버전을 사용하고 있습니다. 코드는 아래와 같습니다.
```html
<!-- MathJax start -->
<script>MathJax = {tex: {inlineMath: [['$', '$'], ['\\(', '\\)']]}};</script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"></script>
<!-- MathJax end -->
```
